net/autossh/src/opnsense/mvc/app/views/VerbNetworks/Autossh/settings.volt

```diff
@@ -0,0 +1,333 @@
+                
+            </div>
+
+        </div>
+    </div>
+</div>
+
+{# include dialogs #}
+{{ partial('layout_partials/base_dialog',['fields':formDialogKeys,'id':'DialogKeys','label':lang._('SSH key')])}}
+{{ partial('layout_partials/base_dialog',['fields':formDialogTunnels,'id':'DialogTunnels','label':lang._('SSH tunnel')])}}
+
+<style>
+    div.type-info div.modal-dialog div.modal-body div.bootstrap-dialog-message {
+        padding: 0.5em;
+        border: 1px solid #999999;
+        font-size: 100%;
+        font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+        word-break: break-all;  
+    }
+</style>
+
+<script>
+    
+    $(document).ready(function() {
+        
+        $("#grid-tunnels").UIBootgrid(
+            {   search:'/api/autossh/tunnels/search',
+                get:'/api/autossh/tunnels/get/',
+                set:'/api/autossh/tunnels/set/',
+                add:'/api/autossh/tunnels/add/',
+                del:'/api/autossh/tunnels/del/',
+                info:'/api/autossh/tunnels/info/',
+                toggle:'/api/autossh/tunnels/toggle/',
+                
+                options:{
+                    ajax: true,
+                    selection: true,
+                    multiSelect: true,
+                    rowCount:[10, 25, 100, -1] ,
+                    formatters:{
+                        tunnel_commands: function(column, row) {
+                            return '<button type="button" class="btn btn-xs btn-default command-info"   data-row-id="' + row.uuid + '"><span class="fa fa-key"></span></button> ' +
+                                   '<button type="button" class="btn btn-xs btn-default command-edit"   data-row-id="' + row.uuid + '"><span class="fa fa-pencil"></span></button> ' +
+                                   '<button type="button" class="btn btn-xs btn-default command-copy"   data-row-id="' + row.uuid + '"><span class="fa fa-clone"></span></button> ' +
+                                   '<button type="button" class="btn btn-xs btn-default command-delete" data-row-id="' + row.uuid + '"><span class="fa fa-trash-o"></span></button>';
+                        },
+                        row_toggle: function (column, row) {
+                            if (parseInt(row[column.id], 2) === 1) {
+                                return '<span style="cursor: pointer;" class="fa fa-check-square-o command-toggle" data-value="1" data-row-id="' + row.uuid + '"></span>';
+                            } else {
+                                return '<span style="cursor: pointer;" class="fa fa-square-o command-toggle" data-value="0" data-row-id="' + row.uuid + '"></span>';
+                            }
+                        },
+                        code_wrap: function(column, row) {
+                            if(row[column.id].length > 0) {
+                                return '<code>' + row[column.id] + '</code>';
```

Comment:
This is one of the places, where you would have to escape if this issue would not exist.

Reply:
This is an excellent catch